using-timescaledb/backup.md

```diff
@@ -6,6 +6,8 @@ physical backups with `pg_basebackup` or another tool, or logical backups with
 `pg_dump` and `pg_restore`. Physical backups may also be used with Write-Ahead Log
 (WAL) archiving to achieve an ongoing backup.
 
+>:WARNING:Timescaledb currently does not support backup and restore for multi-node environments. Care should be taken to ensure third party solutions properly quiesce the environment before backing up.
```

Comment:
```suggestion
>:WARNING:TimescaleDB currently does not natively support a consistent restore point for multi-node environments. Care should be taken to ensure third-party solutions properly quiesce the environment before backing up, so that the backup point used across nodes does not have outstanding transactions.
```